src/common/HTTPRequest.ts

```diff
@@ -296,10 +432,26 @@ export class HTTPRequest {
     // Request interception is not supported for data: urls.
     if (this._url.startsWith('data:')) return;
     assert(this._allowInterception, 'Request Interception is not enabled!');
-    assert(!this._interceptionHandled, 'Request is already handled!');
-    const { url, method, postData, headers } = overrides;
-    this._interceptionHandled = true;
+    assert(
+      !this._interceptionHandled,
+      'Request is already handled! Did you mean to use cooperative_request?'
```

Comment:
Maybe change this to mention that cooperative_request is an event one can subscribe to.